.gitlab-ci.yml

```diff
@@ -0,0 +1,20 @@
+variables:
+  GIT_SUBMODULE_STRATEGY: recursive
+
+cache:
+  key: ${CI_COMMIT_REF_SLUG}
+  paths:
+    - node_modules/
+
+stages:
+  - check
+
+before_script:
+  - node --version
+  - npm install
```

Comment:
We only need dev dependencies. But you can do that later, as I'm going to merge this.